packages/yourloops/components/patient/add-patient-dialog.tsx

```diff
@@ -241,7 +210,7 @@ export const AddPatientDialog: FunctionComponent<AddDialogProps> = ({ onClose, o
         </Button>
         <LoadingButton
           id="patient-list-dialog-add-button-add"
-          disabled={!!errorMessage || !isValidEmail() || teamId.length < 1}
+          disabled={!!errorMessage || !isValidEmail() || selectedTeam.id.length < 1}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this check on selectedTeam usefull ?  Now with team scope we are sure to have a selected team with an id, and in case it's private we don't have access to the dialog

Reply:
I thought the same thing but I was afraid of breaking something... I'll remove the checks on `selectedTeam`, the code will be more understandable